app/controllers/documents_controller.rb

```diff
@@ -39,6 +48,10 @@ def generate_path
 
 private
 
+  def index_params
```

Comment:
This doesn't seem to be used?

Reply:
Good spot. I've combined this with the second comment and now use this method to generate the single params variable to pass to DocumentFilter. I see what you mean that passing a single variable makes sense, especially as we're extracting each one in the initializer already anyway.

Reply:
👍